<?xml version='1.0' encoding='utf-8'?>
<robot xmlns:xacro="http://ros.org/wiki/xacro" name="abb_irb2400"><xacro:include filename="$(find vc_ros_tutorials)/xacro/abb_irb2400_macro.xacro" /><xacro:abb_irb2400 prefix="" /></robot>